internal/setup/setup.go

```diff
@@ -197,34 +199,60 @@ func (r *Release) validate() error {
 		}
 	}
 
-	// Check for glob and generate conflicts.
+	// Check for generate and glob conflicts.
```

Comment:
[Comment for reviewer]: This is based on a nitpick by Ed about reading generate as a verb instead of a noun.